ansible_ai_connect/ai/api/wca/api_key_views.py

```diff
@@ -52,6 +58,7 @@
     IsAuthenticatedOrTokenHasScope,
     IsOrganisationAdministrator,
     IsOrganisationLightspeedSubscriber,
+    IsWCASaaSModelPipeline,
```

Comment:
These endpoints only apply to WCA SaaS.. 

I added a permission to prevent accidental use. 

I also need to cast `MetaData` to `WCASaaSMetaData` to call `get_token(..)` (that only applies to WCA SaaS).